docs/doc_assets/css/styleguide.scss

```diff
@@ -1506,3 +1408,40 @@ $font-light: 300;
     }
   }
 }
+
+// Demo fixed footer
+.demo-footer {
```

Comment:
Whats the demo footer? How does it relate to headers? Why doesn't it need all these separate CSS rules?